docs/data-sources/associated_site.md

```diff
@@ -0,0 +1,72 @@
+---
+# Documentation generated by "gen/generator.go"; DO NOT EDIT.
+# In order to regenerate this file execute `go generate` from the repository root.
+# More details can be found in the [README](https://github.com/CiscoDevNet/terraform-provider-aci/blob/master/README.md).
+subcategory: "Multi-Site"
+layout: "aci"
+page_title: "ACI: aci_associated_site"
+sidebar_current: "docs-aci-data-source-aci_associated_site"
+description: |-
+  Data source for Associated Site
+---
+
+# aci_associated_site #
+
+Data source for Associated Site
+
+## API Information ##
+
+* Class: [fvSiteAssociated](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvSiteAssociated/overview)
+
```

Comment:
missing versions, please check you used the meta file from previously shared link that includes versions

Reply:
all the documentation comments apply to the resource equivalent docs also